docs/get-started/viam.md

```diff
@@ -15,143 +15,172 @@ imageAlt: "/general/understand.png"
 
-Join the [**Viam community**](https://discord.gg/viam) to collaborate during planning and beyond.
+{{<imgproc src="/tutorials/pet-treat-dispenser/preview.png" resize="300x" declaredimensions=true alt="Image of a dog interacting with the smart pet feeder." class="alignleft">}}
 
-## Get started
+Consider a pet feeder which is made up of a webcam and a motor as well as a board with the programming for the device.
 
-A _smart machine_ in Viam consists of at least one computer, typically a [single-board computer](/get-started/installation/), running `viam-server` and communicating with any hardware connected to it by signaling through digital data pins.
-Viam supports devices running **any** 64-bit Linux OS or macOS.
-Here are some of the boards Viam supports:
+TODO: Diagram
 
-{{< board-carousel >}}
-<br>
+With the Viam app, you can configure the motor and webcam as {{< glossary_tooltip term_id="component" text="components" >}}.
 
-{{< imgproc src="/viam/board-viam-server.png" alt="A diagram of a single-board computer running viam-server." resize="270x" class="alignleft" style="max-width:270px" >}}
+### Benefits of the Viam platform
 
-The Viam platform provides a user interface for connecting to and managing machines, the [Viam app](https://app.viam.com/).
+By configuring your machine with Viam, you can now:
 
-To use the Viam platform with your machine, log in to [the app](https://app.viam.com/), create a new machine, and [install](/get-started/installation/) the [`viam-server`](https://github.com/viamrobotics/rdk) binary which:
+{{< table >}}
+{{% tablestep link="/cloud/machines/#control"%}}
+<img src="/get-started/feed.gif" style="max-width:600px" class="fill alignleft">
 
-- Creates, configures, and maintains the machine.
-- Securely handles all communications.
-- Runs drivers, custom code, and any other software.
-- Accepts API requests.
-- Runs services like computer vision, data synchronization, and motion planning.
+**Remotely access and control the machine**
+
+Using the Viam app or the Viam mobile app, you can view the camera feed and turn on the motor to feed your pet.
+
+{{% /tablestep %}}
+{{% tablestep link="/services/data/"%}}
+{{<imgproc src="/get-started/bowl-images.png" class="fill alignleft" resize="800x" style="max-width: 600px" declaredimensions=true alt="Screenshot of food bowl pictures">}}
+**Collect and upload data to the cloud**
+
+You can configure the camera to capture a photo of the food bowl periodically and automatically sync these photos to the cloud.
+
+{{% /tablestep %}}
+{{% tablestep link="/services/ml/"%}}
+{{<imgproc src="/get-started/bowl-label.jpg" class="fill alignright" resize="600x" style="max-width: 200px" declaredimensions=true alt="Screenshot of food bowl pictures with labels">}}
+**Use collected data to build data sets and train and Machine Learning models**
+
+Having captured pictures of the food bowl, you can organize them into a dataset and label each as empty or full.
```

Comment:
I want to take a stab at a different layout (not sure what yet) and try to reduce the word count a bit through these examples. I fear we might lose people partway through.

Reply:
Go for it